content/en/docs/concepts/services-networking/service.md

```diff
@@ -666,6 +666,24 @@ The value of `spec.loadBalancerClass` must be a label-style identifier,
 with an optional prefix such as "`internal-vip`" or "`example.com/internal-vip`".
 Unprefixed names are reserved for end-users.
 
+#### Specifying IPMode of load balancer status {#load-balancer-ip-mode}
+
+{{< feature-state for_k8s_version="v1.29" state="alpha" >}}
+
+Starting as Alpha in Kubernetes 1.29, a [feature gate](https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/) named `LoadBalancerIPMode` allows you to set the `.status.loadBalancer.ingress.ipMode` 
+for a Service with `type` set to `LoadBalancer`.
+The `.status.loadBalancer.ingress.ipMode` specifies how the load-balancer IP behaves, and may be specified only when the `.status.loadBalancer.ingress.ip` field is also specified.
```

Comment:
```suggestion
The `.status.loadBalancer.ingress.ipMode` specifies how the load-balancer IP behaves.
It may be specified only when the `.status.loadBalancer.ingress.ip` field is also specified.
```